### ISSUE TITLE
Applying the Activity Streams schema

### ISSUE THREAD

**Issue by arikan** [appdotnet/api-spec#18]
I think applying the http://activitystrea.ms schema is essential for app.net.

**Comment by simondlr** [comment]
It's already been discussed (or being discussed) at issue #15.

**Comment by mattflaschen** [comment]
I agree it's essential.  it's also one of the things app.net has [specifically committed](http://daltoncaldwell.com/a-response-to-brennan-novak) to implement.  I'm not sure why #15 is closed.  One of these needs to stay open until it's implemented.
